tests/test_color_deconvolution.py

```diff
@@ -2,6 +2,7 @@
 import skimage.io
 
 from histomicstk.preprocessing import color_deconvolution as htk_dcv
+from histomicstk.preprocessing.color_deconvolution.separate_stains_macenko_pca import argpercentile as htk_ap
```

Comment:
```suggestion
from histomicstk.preprocessing.color_deconvolution.separate_stains_macenko_pca import \
    argpercentile as htk_ap
```
Ok - so if I understand correctly, we can suggest a change directly to the pull request